### PR TITLE
Feat/netapp dashboard

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/netapp`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7146
| License          | BSD 3-Clause

## Related
- [ ] advices#32
- [x] https://github.com/ovh/manager/pull/5231

### CODE REVIEW

File: packages/manager/modules/netapp/src/dashboard/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+  "netapp_dashboard_information": "Informations",
+  "netapp_dashboard_information_name": "Nom",
+  "netapp_dashboard_information_id": "ID",
+  "netapp_dashboard_information_region": "Localisation du datacente",
+  "netapp_dashboard_information_performance_level": "Niveau de performance",
+  "netapp_dashboard_information_quota": "Taille du stockage",
+  "netapp_dashboard_information_quota_formatted": "{{ quota }} Go",
+  "netapp_dashboard_configuration": "Configuration",
+  "netapp_dashboard_configuration_status": "Statut",
+  "netapp_dashboard_configuration_status_creating": "En cours de création",
```
Comment (marie-j) [inline packages/manager/modules/netapp/src/dashboard/translations/Messages_fr_FR.json:12]: Waiting content validation on status

File: packages/manager/modules/netapp/src/dashboard/routing.js [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+    },
     resolve: {
       currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
         $state.href($state.current.name, $transition$.params()),
-      dashboardLink: /* @ngInject */ ($state) =>
-        $state.href('netapp.dashboard'),
+      dashboardLink: /* @ngInject */ ($state, $transition$) =>
+        $state.href('netapp.dashboard', $transition$.params()),
+      createVolumeLink: /* @ngInject */ ($state, $transition$) =>
+        $state.href('netapp.dashboard.volume.create', $transition$.params()),
```
Comment (JeremyDec) [inline packages/manager/modules/netapp/src/dashboard/routing.js:17]: State `netapp.dashboard.volumes` will be corresponding to Volumes tab, where `netapp.dashboard.volume` will be the volume detail, so:
```suggestion
        $state.href('netapp.dashboard.volumes.create', $transition$.params()),
```

File: packages/manager/modules/netapp/src/dashboard/component.js [hunk 3]
```diff
@@ -3,9 +3,12 @@ import template from './template.html';
 
 export default {
   bindings: {
-    serviceName: '<',
     currentActiveLink: '<',
     dashboardLink: '<',
+    storage: '<',
+    createVolumeLink: '<',
+    serviceInfos: '<',
```
Comment (JeremyDec) [inline packages/manager/modules/netapp/src/dashboard/component.js:10]: `serviceInfos` is missing in routing

**Review by JeremyDec — CHANGES_REQUESTED** [review]

File: packages/manager/modules/netapp/src/dashboard/component.js (hunk 3 above)
Comment (marie-j) [reply packages/manager/modules/netapp/src/dashboard/component.js:10]: Not needed, forgot to remove it

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**